src/rest_api/routing.clj

```diff
@@ -55,6 +55,10 @@
     [route-spec]
     [route-spec opts]))
 
+(defn swagger-id-field [entity-type]
```

Comment:
Ah Thanks @mgrbyte ! Can I name it `entity-type-alias`? Trying to convey that it's the alias of a class (gene) rather than an instance (WBGene...)